code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -110,6 +110,13 @@
 	ammo_type = /obj/item/ammo_casing/shotgun/tranquilizer
 	materials = list(MAT_METAL=1750)
 
+/obj/item/ammo_box/tranqdarts

```

Comment:
I'm ok with loose pistol/rifle rounds in boxes acting like speedloaders -> mags. Only shotgun shells and bigger should need to be loaded in manually due to their size and encumbrance.